src/Compilers/CSharp/Test/Syntax/Parsing/AsyncParsingTests.cs

```diff
@@ -1681,21 +1681,15 @@ class C
 {
     async partial implicit operator
 ",
-                // (4,11): error CS1553: Declaration is not valid; use '+ operator <dest-type> (...' instead
+                // (4,11): error CS1553: Declaration is not valid; use 'implicit operator <dest-type> (...' instead
                 //     async partial implicit operator
-                Diagnostic(ErrorCode.ERR_BadOperatorSyntax, "partial").WithArguments("+").WithLocation(4, 11),
-                // (4,19): error CS1003: Syntax error, 'operator' expected
+                Diagnostic(ErrorCode.ERR_BadOperatorSyntax, "partial").WithArguments("implicit").WithLocation(4, 11),
+                // (4,36): error CS1031: Type expected
                 //     async partial implicit operator
-                Diagnostic(ErrorCode.ERR_SyntaxError, "implicit").WithArguments("operator").WithLocation(4, 19),
-                // (4,19): error CS1037: Overloadable operator expected
+                Diagnostic(ErrorCode.ERR_TypeExpected, "").WithLocation(4, 36),
+                // (4,36): error CS1003: Syntax error, '(' expected
                 //     async partial implicit operator
-                Diagnostic(ErrorCode.ERR_OvlOperatorExpected, "implicit").WithLocation(4, 19),
-                // (4,28): error CS1003: Syntax error, '(' expected
-                //     async partial implicit operator
-                Diagnostic(ErrorCode.ERR_SyntaxError, "operator").WithArguments("(").WithLocation(4, 28),
-                // (4,28): error CS1041: Identifier expected; 'operator' is a keyword
-                //     async partial implicit operator
-                Diagnostic(ErrorCode.ERR_IdentifierExpectedKW, "operator").WithArguments("", "operator").WithLocation(4, 28),
+                Diagnostic(ErrorCode.ERR_SyntaxError, "").WithArguments("(").WithLocation(4, 36),
```

Comment:
you'll see in a lot of tests a vast decrease in errors in these cases as we recover just fine now.